src/itype.h

```diff
@@ -574,11 +593,17 @@ struct itype {
 
     virtual int charges_to_use() const
```

Comment:
Could be, but it is used a lot in iuse.cpp.

It would be a good idea to only replace it when ammo required takes an argument (ammo required for what? toolbox ammo required for cutting vs toolbox ammo required for repairing), when the legacy vs new would be clearly visible.